doc/task_breakdown.md

```diff
@@ -0,0 +1,52 @@
+# Task Breakdown for SQL Query Builder
+
+## 1. Tables
+- Provide users with the ability to select tables from the database.
```

Comment:
I've asked you to create list of tasks, how you suppose to finish this task? select tables? render all tables with checkboxes?